src/read/cfi.rs

```diff
@@ -218,13 +218,86 @@ impl<R: Reader> ParsedEhFrameHdr<R> {
     }
 }
 
+/// An iterator for `.eh_frame_hdr` section's binary search table.
+/// Each table entry consists of a tuple containing an  `initial_location` and `address`.
```

Comment:
I know this is what the LSB says, but I think we could be clearer about what `initial_location` and `address` mean.